app/routes.py

```diff
@@ -1,2 +1,65 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+from app.models.planet import Planet
 
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+
+@planets_bp.route("", methods=["GET", "POST"])
```

Comment:
Consider separating each request method into its different functions. This makes testing the endpoints much easier to do and follows the single responsibility principle.